lib/src/Dropdown/Dropdown.tsx

```diff
@@ -206,12 +206,12 @@ export const Dropdown: React.FC<DropdownProps> = React.forwardRef(({ wrapperProp
                 document.removeEventListener("click", detectBlur);
                 window.removeEventListener("wheel", handleScroll);
             };
-        }, [show]);
+        }
+    }, [show]);
 
-        React.useEffect(() => {
-            setLabel((Array.isArray(props.value) ? props.value.join(", ") : props.value) || props.placeholder);
-        }, [props.value]);
-    }
+    React.useEffect(() => {
+        !isMobile && setLabel((Array.isArray(props.value) ? props.value.join(", ") : props.value) || props.placeholder);
```

Comment:
```suggestion
        !isMobile && setLabel((Array.isArray(props.value) ? props.value?.join(", ") : props.value) || props.placeholder);
```

Reply:
`Array.isArray` will not return true if `props.value` is `null` or `undefined`. It's already checked, no need for another optional chaining